features/step-definitions/task.1.js

```diff
@@ -0,0 +1,71 @@
+const { Given, When, Then } = require('@cucumber/cucumber');
+const HomePage = require('../pageobjects/home.page');
+const AddRemoveElementPage = require('../pageobjects/add.remove.element.page')
+const DynamicLoadingPage = require('../pageobjects/dynamic.loading.page')
+Given(/^I am on the Add Remove Elements page$/, async () => {
+    await HomePage.open();

```

Comment:
This opens homepage, not add/remove elements page. Either change the step title or the steps themselves